src/plugins/intel_cpu/src/node.h

```diff
@@ -95,6 +97,12 @@ class NodeDesc {
     impl_desc_type implementationType;
 };
 
+struct NodeRuntime {
+    MultiCache paramsCache;
+    ScratchPad scratchPad;
+    NodeRuntime(size_t rtCacheCapacity, dnnl::engine eng) : paramsCache(rtCacheCapacity), scratchPad(eng) {}
+};
```

Comment:
Honestly, I can not get on why it is good to joint cache and scratchpad in one structure. From the usage perspective, we extended the node interface with an additional call on getting the scratchpad memory. It looks like node scratchpad is a separate entity from the cache, I guess it should be handled separately. I suspect that this structure was introduced since the cache and the scratchpad are shared across all nodes in the graph, but such joining is rather brings more difficulties. What if we want to use scratchpad but not cache? What if we use the cache across all the graphs(streams), which we actually plan to do?
Also what if decide to introduce a mode where we disable scratchpad usage? In case of cache we can set 0 capacity.

Reply:
OK, Splitted